alias.go

```diff
@@ -27,12 +27,16 @@ type WithOption = option.Option
 var (
 	WithChroot       = option.WithChroot
 	WithSnapshot     = option.WithSnapshot
-	WithAlterter     = option.WithAlerter
-	WithNullAlterter = option.WithNullAlerter
+	WithAlerter      = option.WithAlerter
+	WithNullAlerter  = option.WithNullAlerter
 	// match the existing environ variable to minimize surprises
 	WithDisableWarnings = option.WithNullAlerter
 	WithDisableTools    = option.WithDisableTools
 	WithPathOverrides   = option.WithPathOverrides
+	// Deprecated: WithAlterter is misspelled, Instead, use WithAlerter.
+	WithAlterter      = option.WithAlerter
+	// Deprecated: WithNullAlterter is misspelled. Instead, use WithNullAlerter.
+	WithNullAlterter  = option.WithNullAlerter
```

Comment:
I actually don't think we need to keep these around. Let's just fix the misspelled ones.